LupusNet/ETL/Gladel2.0/runETL.sh

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+set -e
+
+#############
+## RUN ETL ##
+#############
+REGISTRY_ETL_RUNNER=harbor.honeur.org
+REPOSITORY_ETL_RUNNER=library
+IMAGE_ETL_RUNNER=etl-runner
+VERSION_ETL_RUNNER=1.1.4
+TAG_ETL_RUNNER=$VERSION_ETL_RUNNER
+
+LOG_FOLDER_HOST=${PWD}/log
+DATA_FOLDER_HOST=${PWD}/data
+QA_FOLDER_HOST=${PWD}/qa
+
+echo "Pull ETL runner image"
+docker pull $REGISTRY_ETL_RUNNER/$REPOSITORY_ETL_RUNNER/$IMAGE_ETL_RUNNER:$TAG_ETL_RUNNER
+
+REGISTRY=harbor.lupusnet.org
+REPOSITORY=etl-gladel-2.0
+
+# Log into Harbor
+echo "Login into Harbor (please enter your email address and Harbor CLI secret if needed)"
+docker login $REGISTRY
```

Comment:
Docker login is not needed because the ETL runner is a public image and the ETL is pulled using the CLI secret configured in the local confirguration.
Please remove docker login as it can confuse the end user if the password is requested by the run script